### DISCUSSION TITLE
不会自建路由，有没有可以付费使用的路由？

### DISCUSSION THREAD

**Discussion by Chris-zixuan** [DIYgod/RSSHub#11867] [Q&A]
想用RSS，但是示例的路由地址都抓取不到，看到说可以自建路由。无奈不是科班生 不太懂。。。
有没有能够付费使用的路由地址？

**Comment by Chris-zixuan** [comment]
鹅。。表达有误 不是路由

例如这个路由：https://rsshub.app/hupu/nba
我不开梯子是抓取不到内容的，开了梯子就可以。教程里好像说过是因为  RSShub  这个被屏蔽了，需要自建站点。
想问的是有没有可以付费使用的，国内直连的站点域名。

**Comment by chenchienchen** [comment]
https://xn--4gq62f52gdss.com/#/register?code=xEjb5x4y这个梯子11快钱一年亲测便宜好用，真的！

**Reply by Chris-zixuan** [reply]
有点坑啊哥们。。我注册后基本没几个能用的。。我倒是有个更好的你要不，一个月10块钱

**Reply by chenchienchen** [reply]
我一直再用啊我这个，嘎嘎好用啊

**Reply by Chris-zixuan** [reply]
我测速，节点一片红啊哥们

**Reply by chenchienchen** [reply]
我现在用的他自动选择的节点两百多延迟

**Reply by yangsaipp** [reply]
我正在找梯子，能不能分享下？最好是按月的

**Comment by xabyo** [comment]
https://rsshub.rssforever.com
https://rss.qiuyuair.com
https://rss.injahow.cn
https://rss.feiyuyu.net
https://rss.shab.fun
https://rss.itggg.cn
https://rsshub.uneasy.win
https://rss.injahow.cn
https://rsshub.anyant.xyz
